js/src/setup-mc/setup-stepper/setup-paid-ads/setup-paid-ads.js

```diff
@@ -129,16 +120,14 @@ export default function SetupPaidAds() {
 			campaign_form_validation: 'unknown',
 		};
 
-		if ( showPaidAdsSetup ) {
-			const selector = select( STORE_KEY );
-			const billing = selector.getGoogleAdsAccountBillingStatus();
+		const selector = select( STORE_KEY );
+		const billing = selector.getGoogleAdsAccountBillingStatus();
 
-			merge( eventProps, {
-				opened_paid_ads_setup: 'yes',
-				billing_method_status: billing?.status,
-				campaign_form_validation: paidAds.isValid ? 'valid' : 'invalid',
-			} );
-		}
+		merge( eventProps, {
+			opened_paid_ads_setup: 'yes',
```

Comment:
Now that we're removing the possibility of skipping without opening the ads setup, this event prop should be removed from the whole component and removed from the inline docs for `gla_onboarding_complete_button_click` above.

I think we could also avoid the need to merge event props here and just set them directly when `eventProps` is initialized above.

Reply:
Good catch and makes sense @joemcgill . I've updated the PR. Can you kindly take a look again please?